src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -416,12 +417,7 @@ export class SQLFuture extends Disposable implements FutureInternal {
 		return this.doneDeferred.promise;
 	}
 
-	public handleDone(err?: Error): void {
-		this.handleDoneAsync(err);
-		// TODO we should reject where some failure happened?
-	}
-
-	private async handleDoneAsync(err?: Error): Promise<void> {
+	public async handleDone(err?: Error): Promise<void> {
```

Comment:
@chlafreniere - Currently this parameter is never being used, although we pass it in here https://github.com/microsoft/azuredatastudio/blob/master/src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts#L299

Should we just remove it - or do you want to handle the error in some way?

Reply:
Let's just leave this for now and have a think about what's best here. This particular case is harmless and we may need a generic approach here.